popup/components/EntryList.tsx

```diff
@@ -81,44 +81,61 @@ export const EntryList = ({ entries, noEntriesOverlay }: Props) => {
         />
         <Group align="center" w="100%" position="apart">
           <Group align="center" spacing={0}>
-            <ActionIcon
-              sx={(theme) => commonActionIconSx({ theme, disabled: selectedEntryIds.size === 0 })}
-              onClick={
-                selectedEntryIds.size === 0
-                  ? undefined
-                  : () =>
+            <Tooltip
+              label={
+                <Text fz="xs">Favorite</Text>
+              }
+            >
```

Comment:
Oh I almost forgot. Can you disable all the tooltips when their corresponding buttons are also disabled?

```suggestion
            <Tooltip
              label={
                <Text fz="xs">Favorite</Text>
              }
              disabled={selectedEntryIds.size === 0}
            >
```

Reply:
Sure, I've disabled the tooltip for those action buttons when no one is selected.